<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataSettingsWidget</class>
 <widget class="QWidget" name="DataSettingsWidget">
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Cơ sở dữ liệu (Database)</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Đường dẫn file:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="dbPathLineEdit">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="changeDbPathButton">
          <property name="text">
           <string>Thay đổi...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Tự động lưu (Auto-save)</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="autoSaveCheckBox">
        <property name="text">
         <string>Bật tự động lưu cho tab &quot;Nhập Dữ Liệu&quot;</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Lưu mỗi:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="autoSaveIntervalSpinBox">
          <property name="suffix">
           <string> phút</string>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Sao lưu &amp; Phục hồi (Backup &amp; Restore)</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="backupButton">
        <property name="text">
         <string>Sao lưu ngay...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="restoreButton">
        <property name="text">
         <string>Phục hồi từ bản sao lưu...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
</ui>
